README.md

```diff
@@ -17,6 +17,8 @@ Easiest way to get the perfect repository.
 * [gorm](https://github.com/go-gorm/gorm), [docs](https://pkg.go.dev/github.com/avito-tech/go-transaction-manager/gorm) (Go 1.16)
 * [mongo-go-driver](https://github.com/mongodb/mongo-go-driver), [docs](https://pkg.go.dev/github.com/avito-tech/go-transaction-manager/mongo) (Go 1.13)
 * [go-redis/redis](https://github.com/go-redis/redis), [docs](https://pkg.go.dev/github.com/avito-tech/go-transaction-manager/redis) (Go 1.17)
+* [pgx_v4](https://github.com/jackc/pgx), [docs](https://pkg.go.dev/github.com/avito-tech/go-transaction-manager/pgx) (Go 1.16)
```

Comment:
https://github.com/jackc/pgx/tree/v4

https://pkg.go.dev/github.com/avito-tech/go-transaction-manager/pgxv4

Reply:
I will do it